econml/dml.py

```diff
@@ -529,7 +557,7 @@ def __init__(self,
                          random_state=random_state)
 
     # override only so that we can update the docstring to indicate support for `StatsModelsInference`
-    def fit(self, Y, T, X=None, W=None, *, sample_weight=None, sample_var=None, groups=None, inference=None):
+    def fit(self, Y, T, X=None, W=None, sample_weight=None, sample_var=None, groups=None, inference='auto'):
```

Comment:
Continue to require that the final arguments are passed only by name.
```suggestion
    def fit(self, Y, T, X=None, W=None, *, sample_weight=None, sample_var=None, groups=None, inference='auto'):
```